net/FlatBuffers/Struct.cs

```diff
@@ -19,9 +19,9 @@ namespace FlatBuffers
     /// <summary>
     /// All structs in the generated code derive from this class, and add their own accessors.
     /// </summary>
-    public abstract class Struct
+    public struct Struct
     {
-        protected int bb_pos;
-        protected ByteBuffer bb;
+        public int bb_pos;
```

Comment:
As these are public, we may want to adopt a more idiomatic C# convention which is to use PascalCase names for members.